src/kernel/disconnected_transactions.cpp

```diff
@@ -51,9 +51,8 @@ size_t DisconnectedBlockTransactions::DynamicMemoryUsage() const
     iters_by_txid.reserve(iters_by_txid.size() + vtx.size());
     for (auto block_it = vtx.rbegin(); block_it != vtx.rend(); ++block_it) {
         auto it = queuedTx.insert(queuedTx.end(), *block_it);
-        auto [_, inserted] = iters_by_txid.emplace((*block_it)->GetHash(), it);
-        assert(inserted); // callers may never pass multiple transactions with the same txid
```

Comment:
I don't think you meant to remove this? It reverts f4254e209801d6a790b5f0c251c0b32154a4e3cc

Reply:
interactive rebase mistake, fixed in latest push.